assets/_scss/elements/_list.scss

```diff
@@ -1,29 +1,35 @@
-// Make lists unstyled by default
 ul, ol {
```

Comment:
I added a rule in our styleguide that selectors separated by commas only have to be on new lines if they are more then 5 chars, so this would be fine. Let me know if you want them to always have newlines, so like this:

```
ul,
ol {
...

```